src/OrbitGl/AccessibleCaptureViewElement.cpp

```diff
@@ -18,33 +18,31 @@ const orbit_accessibility::AccessibleInterface* AccessibleCaptureViewElement::Ac
 }
 orbit_accessibility::AccessibilityRect AccessibleCaptureViewElement::AccessibleRect() const {
   CHECK(capture_view_element_ != nullptr);
-  GlCanvas* canvas = capture_view_element_->GetCanvas();
-  CHECK(canvas != nullptr);
-  const Viewport& viewport = canvas->GetViewport();
+  const Viewport* viewport = capture_view_element_->GetViewport();
```

Comment:
It's probably more consistent with the rest of our code like this, I'm fine with changing it. For the record, I do think we should use references more throughout the code to indicate that parameters / return values cannot be null. For input parameters, this is also recommended by the Style Guide: https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs "Non-optional input parameters should usually be values or const references"

Reply:
After done with my review, I guess it is fine. The alternative would be to store a copy of the viewport, which I'd agree of being not better.

In general, I agree what what you said. Pointers should not be our default.

Reply:
I was investigating a little bit. Another option (not 100% sure) is to have a pointer in TimeGraph and a const reference in the rest of the CaptureViewElements. In this case we are only being able to modify the Viewport in TimeGraph (SetWorldTopLeft). What do you think?

Reply:
Since `TimeGraph` is inheriting from `CaptureViewElement`, it feels weird to have this behave differently. My argument is also more about reference vs. pointer, not so much about the `const` - I think it's reasonable that `CaptureViewElements` are allowed to modify the viewport. My main argument is, if something is expected to be non-null, I prefer references over pointers. Whenever we have a pointer, we should usually assume that this can be null.

Reply:
I supposed the option was pointers vs const ref. Are we using non-const references?

Reply:
After a few discussions, I will leave this at it is now.